go/vt/vtgate/planbuilder/testdata/dml_cases.json

```diff
@@ -2648,6 +2648,30 @@
     },
     "skip_e2e": true
   },
+  {
+    "comment": "insert using last_insert_id with argument (already an e2e test for this plan)",
+    "query": "insert into unsharded values(last_insert_id(789), 2)",
+    "plan": {
+      "QueryType": "INSERT",
+      "Original": "insert into unsharded values(last_insert_id(789), 2)",
+      "Instructions": {
+        "OperatorType": "Insert",
+        "Variant": "Unsharded",
+        "Keyspace": {
+          "Name": "main",
+          "Sharded": false
+        },
+        "TargetTabletType": "PRIMARY",
+        "FetchLastInsertID": true,
+        "Query": "insert into unsharded values (last_insert_id(789), 2)",
+        "TableName": "unsharded"
+      },
+      "TablesUsed": [
+        "main.unsharded"
+      ]
+    },
+    "skip_e2e": true
```

Comment:
this should not be skipped

Reply:
it needs to be skipped. we keep getting primary key violations if we don't skip it